package.json

```diff
@@ -25,6 +25,7 @@
     "email:dev": "email dev -p 3001"
   },
   "dependencies": {
+    "@apollo/server": "^4.11.0",
```

Comment:
We need to decouple grapqhl from the platform and add another library may make this work difficult.

Reply:
This was already a project dependency for running GraphQL, but it wasn’t listed in the package.json